init.js

```diff
@@ -0,0 +1,28 @@
+let tetris = require('./tetris');
+module.exports = {
+    init: function () {
+        let fieldSIze = { x: 5, y: 5 };
+        let newTetris = new tetris.Tetris(fieldSIze, stopTick());
+        let tick = setInterval(newTetris.tick, 1000);
+        keyboardHandler(event.keyCode, newTetris)
+        function keyboardHandler(keyCode, tetris) {
```

Comment:
incorrect argument list for event handler

Reply:
Still incorrect listener singnature
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Parameters